.github/workflows/molecule.yml

```
@@ -26,6 +26,7 @@ jobs:
           - centos8
           - centos7
           - rockylinux8
+          - almalinux8
```

```suggestion
          # - almalinux8  #TODO
```
Action Molecule / test (almalinux8) failed, because geerlingguy/docker-almalinux8-ansible, repository does not exist. \
See https://hub.docker.com/u/geerlingguy/

For now, I can suggest adding tests for AlmaLinux to the TODO list.